airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -686,3 +688,69 @@ class ReviewComments(IncrementalGithubStream):
 
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return f"repos/{stream_slice['repository']}/pulls/comments"
+
+
+# Reactions streams
+
+
+class ReactionStream(GithubStream, ABC):
+
+    parent_key = "id"
+    fields_to_minimize = ("user",)
```

Comment:
we shouldn't minimize this since it may contain new users not in any other streams 
```suggestion
```